model/model.py

```diff
@@ -0,0 +1,49 @@
+"""Define the main MeshGraphNet model"""
+
+import numpy as np
+import torch
+
+from meshnets.datatypes.graph import Graph
+from meshnets.model.encoder import GraphEncoder
+from meshnets.model.decoder import GraphDecoder
+from meshnets.model.processor import GraphProcessor
+
+
+class MeshGraphNet(torch.nn.Module):
+    """MeshGraphNet model.
+    
+    It is composed of a GraphEncoder, a GraphProcessor and a GraphDecoder.
+    It takes as input Graph objects, encodes their node and mesh features
+    to a latent representation, apply message passing steps
+    on the latent Graph objects, and decodes the output features
+    at each node."""
+
+    def __init__(self, node_feats_size, mesh_feats_size, output_size,
```

Comment:
What do you mean by "feats" it is to reference the features?

Do not be afraid to be as descriptive as possible. "node_features_size" is a wonderful name for the variable and a great way to document what you are doing.

Is the `num_mlp_layers` equal for both the edges and nodes MLP encoder?

Reply:
Yes feats refers to features. Thanks for the suggestion, I will make the arguments more explicit.

From the original implementation, num_mlp_layers is identical for both edges and nodes MLP encoder, as well as for the decoder. The latent size is also identical for the edge and node features.